### ISSUE TITLE
Not Working in Latest cPanel DNSONLY v78.0.12

### ISSUE THREAD

**Issue by donsony** [letsencrypt-cpanel/letsencrypt-cpanel-dnsonly#10]
service: cpanel
  service_description: Calendar, cPanel, WebDisk, Webmail, and WHM Services
metadata:
  command: install_service_ssl_certificate
  reason: OK
  result: 1
  version: 1
cpanel: unrecognized service
20190225162313 [ERROR] acme.hooks: hook script: /usr/libexec/acme/hooks/certificate-hook.sh: exit status 1

**Comment by alexzorin** [comment]
Is this a new server or did it start happening after an upgrade to 78?

Could you also confirm you are on at least 78.0.12? There are some pretty suspicious bugfixes relating to this listed @ https://documentation.cpanel.net/display/CL/78+Change+Log .

**Comment by donsony** [comment]
Its a newly whipped up server.

Yes, version is v78.0.12

**Comment by alexzorin** [comment]
Alright. I installed a fresh server:

```
# whmapi1 version
---
data:
  version: 11.78.0.12
```

Tried installing `letsencrypt-cpanel-dnsonly`, which succeeded fine (visible at https://alien-spirit.bnr.la:2087/ for now, I'll remove it soon).

It's highly suspicious to me that cPanel would remove `cpanel` as an SSL service component with literally no warning whatsoever. It is a breaking API change and they should not do that.

I think your installation is busted in some way (partial failure maybe?).

Can you confirm the following:

```
# whmapi1 fetch_service_ssl_components | grep "service:"
      service: exim
      service: dovecot
      service: cpanel
```

and also:

```
# systemctl status cpanel
● cpanel.service - cPanel services
   Loaded: loaded (/etc/systemd/system/cpanel.service; enabled; vendor preset: disabled)
   Active: active (running) since Tue 2019-02-26 07:53:12 AEDT; 4s ago
```

Can I also ask what distro and version you are using?

**Comment by alexzorin** [comment]
I've released 0.1.1 which uses the WHM API to restart services instead of `/sbin/service`, which should work around whatever issue your server is having with that method.

You should be able to `yum clean all` and then install the newer version. LMK if it's still broken after that.
